### ISSUE TITLE
03-grid-layout-3: remove empty declaration blocks

### ISSUE THREAD

**Issue by thatblindgeye** [TheOdinProject/css-exercises#170]
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `location for request: brief description of request` format, e.g. `Foundations exercises: Add exercise for XYZ`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

Currently both the `style.css` and the pre-solution portion of the `solution.css` files for this exercise include empty declaration blocks, e.g:

```css
.logo {
}
```

These empty declaration blocks are not used in other exercises in this repo and should be removed for consistency, and to avoid leading users towards a specific way of doing things (since this repo emphasizes that there are multiple ways of doing things).

**2. Acceptance Criteria:**

- [ ] Remove empty declaration blocks in the `style.css` and `solution.css` files

**3. Additional Information:**

**Comment by dm-murphy** [comment]
> 

@thatblindgeye 

That was set up purposefully to give learners the selectors they needed to fill in, since some of these weren't obvious. As one of the Hints points out: "You only need to add to the current CSS selectors"

If these blank selectors are removed from the exercise then that hint also needs to be removed and ideally replaced with a note that the learner will need to add missing selectors.

**Comment by thatblindgeye** [comment]
@dm-murphy Ah, I see what you mean. Do you feel strongly about keeping those empty declaration blocks in?

**Comment by dm-murphy** [comment]
> @dm-murphy Ah, I see what you mean. Do you feel strongly about keeping those empty declaration blocks in?

@thatblindgeye No, I'm certainly open to changing it if that would help. Did any learners report being confused by the empty declarations? 

I think giving them ahead of time makes this exercise more about practicing the right CSS Grid properties and less about designing a layout. This was really a warmup before the follow-up Dashboard project that has them do everything from scratch.

We could also try removing them and monitor any learner difficulties/questions that come up on Discord. Whatever you think is best!

**Comment by thatblindgeye** [comment]
@dm-murphy  I don't believe there have been any issues raised by users about it. For me it was just more about consistency and feeling users should be able to come up with the selectors for the exercise by the time they reach it. I'd say it could be worth removing the empty declaration blocks and rewording the exercise README to reflect that change, or add some other helpful hints that can sort of replace those empty blocks.

**Comment by thatblindgeye** [comment]
@dm-murphy If you're feeling up for it at some point (doesn't have to be immediate) we could have another PR made to reword those hints.

**Comment by dm-murphy** [comment]
> @dm-murphy If you're feeling up for it at some point (doesn't have to be immediate) we could have another PR made to reword those hints.

Sounds good @thatblindgeye! I can take care of the rewording with another PR.

**Comment by dm-murphy** [comment]
@thatblindgeye We should be good to go on both PR #173 and PR #175 that would close out this issue. I also just merged PR #183 which had an unrelated fix to the same exercise. I can review both open PRs and close this all out if that's good with you, unless there was anything else you'd like done for this exercise. Thanks! :-)
